config/default/manager_auth_proxy_patch.yaml

```diff
@@ -18,6 +18,7 @@ spec:
         - "--v=10"
         ports:
         - containerPort: 8443
+          protocol: TCP
```

Comment:
This change is going to require `make bundle` as well. I suggest moving this commit to before the `make bundle` commit.

Reply:
I reordered the commits, but nothing changed in the csv after running make bundle again. Should I have expected a change or was this change solely for proper commit ordering?

Reply:
If no change happened, that makes sense. Looks like this isnt being used at the moment by the operator.
In general all changes to config/ should be followed up by a make bundle.

Reply:
Makes sense! Thanks